docs/usage.md

```diff
@@ -132,10 +132,14 @@ $BootstrapFile = "${Env:Temp}\$(${BootstrapUrl}.split('/')[-1])"
 (New-Object System.Net.WebClient).DownloadFile("$BootstrapUrl", "$BootstrapFile")
 
 # Install python
-& "$BootstrapFile" -PythonUrl "$PythonUrl" -Verbose -ErrorAction Stop
+$params = "`"$BootstrapFile`" -PythonUrl `"$PythonUrl`" -Verbose -ErrorAction Stop"
+Start-Process powershell -Argument $params -NoNewWindow -Wait
+$env:Path = "$env:Path;$env:ProgramFiles\Python36\Scripts\;$env:ProgramFiles\Python36\"
 
 # Install watchmaker
-pip install --index-url="$PypiUrl" --upgrade pip setuptools watchmaker
+pip install --index-url="$PypiUrl" --upgrade pip setuptools boto3
+pip download --dest $env:windir\Temp watchmaker
```

Comment:
Need to maintain the `--index-url="$PypiUrl"` in this command.

Reply:
Reverted to the original pip line and retested.  Worked end to end so the only change that needs to be made is to install Python using Start-Process versus the Call Operator.